static/app/views/settings/project/projectFilters/projectFiltersChart.tsx

```diff
@@ -42,6 +42,8 @@ const STAT_OPS = {
   'release-version': {title: t('Release'), color: theme.purple200},
   'web-crawlers': {title: t('Web Crawler'), color: theme.red300},
   'filtered-transaction': {title: t('Health Check'), color: theme.yellow400},
+  'react-hydration-errors': {title: t('Hydration Errors'), color: theme.outcome.filtered},
+  'chunk-load-error': {title: t('Chunk Load Errors'), color: theme.outcome.filtered},
```

Comment:
How are the colors defined? 🧐 For now, I've used the same color used in the "Filtered" category on the status page.